0.Preprocess_Data/2.data_aggregation.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if (aggregation == True) and (nomic == True):
</pre><p>Consider making functions and/or creating separate analysis to accommodate this change. This might also help with keeping the code DRYness in the same analysis </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>